addons/web_editor/static/src/components/media_dialog/file_selector.xml

```diff
@@ -57,9 +57,10 @@
             validateUrl="validateUrl"
             multiSelect="props.multiSelect"/>
         <t t-call="{{ constructor.attachmentsListTemplate }}"/>
-        <div name="load_more_attachments" class="mt-4 text-center mx-auto o_we_load_more" t-ref="load-more-button">
-            <button t-if="canLoadMore" class="btn btn-primary o_load_more" type="button" t-on-click="handleLoadMore">Load more...</button>
-            <div t-if="hasContent and !canLoadMore" class="mt-4 o_load_done_msg">
+        <div name="load_more_attachments" class="position-fixed pt-3 text-center mx-auto o_we_load_more" t-ref="load-more-button">
+            <div t-if="canScroll" class="d-flex align-items-center mx-auto btn btn-primary rounded-circle fa fa-chevron-down o_scroll_attachments" t-on-click="handleScrollAttachments"/>
```

Comment:
I'm not 100% sure but I think you could use `oi oi-chevron-down` here to be consistent with the style of the other "edit mode" options